docs/releases/5.1.txt

```diff
@@ -275,6 +275,21 @@ Database backend API
 This section describes changes that may be needed in third-party database
 backends.
 
+* The new flag ``supports_update_conflicts_with_condition`` has been added to the database
+  backend features. This flag indicates whether a database backend supports
+  conditional updates in bulk operations like ``bulk_create``. By default, this
+  is set to ``False``. Backends that support this feature should override this
+  flag in their respective ``features.py`` file.
+
+* The ``Excluded`` class has been introduced to reference field values in the proposed data 
+  for bulk update operations. It is particularly useful in conjunction with the ``condition`` 
+  parameter of ``bulk_create`` to specify conditions based on the proposed values for insertion. 
+  Third-party database backends may need to consider this new class when implementing support 
+  for ``supports_update_conflicts_with_condition``.
+
+* A new feature flag ``supports_excluded_expression`` has been added. This flag indicates whether
+  the backend supports the ``Excluded`` class during an ``bulk_create`` operation.
+  
```

Comment:
This is not what I was expecting for the release notes 🤔 
I think you want something in the Models section about the new `condition` parameter in the `bulk_create_method`. 
I'm actually not sure we document the database feature flags in the release notes (I don't expect anything to appear here where we don't have related documentation you could link to) so I think remove the flags.

I don't think any of this is backwards incompatible?